src/integration-test/tests/citizen/defence/full_admission_test.ts

```diff
@@ -0,0 +1,37 @@
+import I = CodeceptJS.I
+
+import { createClaimData } from 'integration-test/data/test-data'
+import { PartyType } from 'integration-test/data/party-type'
+import { PaymentOption } from 'integration-test/data/payment-option'
+
+import { Helper } from 'integration-test/tests/citizen/endToEnd/steps/helper'
+import { DefenceSteps } from 'integration-test/tests/citizen/defence/steps/defence'
+
+const helperSteps: Helper = new Helper()
+const defenceSteps: DefenceSteps = new DefenceSteps()
+
+Feature('Fully admit all of the claim')
+
+Before(function* (I: I) {
+  const claimantEmail: string = yield I.createCitizenUser()
+  const defendantEmail: string = yield I.createCitizenUser()
+
+  const claimData: ClaimData = createClaimData(PartyType.INDIVIDUAL, PartyType.INDIVIDUAL)
+  const claimRef: string = yield I.createClaim(claimData, claimantEmail)
+
+  yield helperSteps.enterPinNumber(claimRef, claimantEmail)
+  helperSteps.linkClaimToDefendant(defendantEmail)
+  helperSteps.startResponseFromDashboard(claimRef)
```

Comment:
It seems they fixed the issue. Screenshot is taken and indicates the issue happened in the hook.

The only difference is that test execution stops when first issue occurred which means that if grep expression is picking for example 3 tests behind the same hook then failure in the hook of the first test case will stop other tests from being fired.

I guess the assumption here is that since all tests share the same prep logic if one failed there is hight risk that the other will fail as well.

I would be keep to leave it as is and see whether we can have any issues with it. Alternatively I can extract the logic into method and call that method in every tests.

Reply:
<img width="826" alt="screen shot 2018-06-25 at 17 46 59" src="https://user-images.githubusercontent.com/1724691/41864101-aa6d3b7c-78a0-11e8-9449-d73c0e0e7dc2.png">